src/js/App/Header/Header.js

```diff
@@ -20,7 +18,7 @@ export const Header = ({ logoClassName }) => {
       <a href="./" className={`ins-c-header-link${logoClassName ? ` ${logoClassName}` : ''}`}>
         <Logo />
       </a>
-      {user && isFilterEnabled && <AppFilter />}
+      <AppFilter />
```

Comment:
Do we want to show it without `user`?

Reply:
@karelhala I don't think so. Maybe its for some special openshift use case?

Reply:
It was a mistake, the filter should be visible only for logged in users.